### ISSUE TITLE
RouteJs Updates?

### ISSUE THREAD

**Issue by barbedCoil** [Daniel15/RouteJs#62]
Checking to see if this is still going to be maintained?
Not pushing just wanting to know if I need to look for a different solution for Asp.Net Core 2.x or if there is something that is better to work with this version of Asp.Net?
Great work, this project has been a lifesaver!

**Comment by Daniel15** [comment]
Yeah, this is still maintained as long as ASP.NET provides some way to get
a list of all the routes. I haven't checked if any changes are required for
ASP.NET Core 2.x... Does it not work for you?

Sent from my phone.

On Feb 12, 2018 8:35 AM, "trh" <notifications@github.com> wrote:

Checking to see if this is still going to be maintained?
Not pushing just wanting to know if I need to look for a different solution
for Asp.Net Core 2.x or if there is something that is better to work with
this version of Asp.Net?
Great work, this project has been a lifesaver!

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/Daniel15/RouteJs/issues/62>, or mute the thread

**Comment by barbedCoil** [comment]
Good to know and thanks for answering, I have noticed quite a few components in our company's dev silo that are not supporting ASP.NET Core let alone 2.x, hence the question.

Yes, so far it does work with exception to this issue noted here: https://github.com/Daniel15/RouteJs/issues/61
